lua/astrocommunity/ai/llm-nvim/init.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  "Kurama622/llm.nvim",
+  dependencies = { "nvim-lua/plenary.nvim", "MunifTanjim/nui.nvim" },
+  cmd = { "LLMSessionToggle", "LLMSelectedTextHandler", "LLMAppHandler" },
+  keys = {
+    { "<leader>ac", mode = "n", "<cmd>LLMSessionToggle<cr>" },
+  },
```

Comment:
```suggestion
  keys = {
    { "<leader>ac", mode = "n", "<cmd>LLMSessionToggle<cr>" },
  },
```

This should be AstroCore and fix the `<Leader>` casing

Reply:
Man I am rusty.